stable/rabbitmq-ha/README.md

```diff
@@ -81,7 +81,7 @@ and their default values.
 | `definitionsSource`                            | Use this key within an existing secret to reference the definitions specification | `"definitions.json"` |
 | `image.pullPolicy`                             | Image pull policy                                                                                                                                                                                     | `Always` if `image` tag is `latest`, else `IfNotPresent`   |
 | `image.repository`                             | RabbitMQ container image repository                                                                                                                                                                   | `rabbitmq`                                                 |
-| `image.tag`                                    | RabbitMQ container image tag                                                                                                                                                                          | `3.7-alpine`                                               |
+| `image.tag`                                    | RabbitMQ container image tag                                                                                                                                                                          | `3.7.11-alpine`                                            |
```

Comment:
Can you switch to the latest 3.7.12-alpine?

Reply:
OK, sure